Source/Xamarin/Prism.Forms.Tests/Services/PageDialogServiceFixture.cs

```diff
@@ -20,68 +20,273 @@ public PageDialogServiceFixture()
         [Fact]
         public async Task DisplayActionSheetNoButtons_ShouldThrowException()
         {
-            var service = new PageDialogServiceMock("cancel", _applicationProvider);
-            var argumentException = await Assert.ThrowsAsync<ArgumentException>(() => service.DisplayActionSheetAsync(null, null));
-            Assert.Equal(typeof(ArgumentException), argumentException.GetType());
+            var service = new PageDialogServiceMock( "cancel", _applicationProvider );
+            var argumentException = await Assert.ThrowsAsync<ArgumentException>( () => service.DisplayActionSheetAsync( null, null ) );
+            Assert.Equal( typeof( ArgumentException ), argumentException.GetType() );
+        }
+
+        #region Tests using Action
+
+        private async Task DisplayActionSheet_PressButton_UsingAction( string text )
```

Comment:
You seem to have an odd setting for brackets and spacing? By default there's no spacing behind ( and before ).

```
private async Task DisplayActionSheet_PressButton_UsingAction(string text)
{
    var service = new PageDialogServiceMock(text, _applicationProvider);
```

This triggers a lot of 'changed code' at the bottom of the file, which isn't changed at all.